website/docs/concepts/step-function/event-sources.md

```diff
@@ -0,0 +1,52 @@
+# Event Sources
+
+AWS Step Functions sends [Event Bus events](https://docs.aws.amazon.com/step-functions/latest/dg/cw-events.html) for each machine execution. Functionless provides easy access to them through Event Bus [Event Sources](../event-bridge/event-sources).
```

Comment:
maybe you can link to this page from the event bridge event-sources page

Reply:
I do, there is a table at the bottom.
![image](https://user-images.githubusercontent.com/289213/171063121-bb3e011c-f139-4a71-8e97-275f15aedab7.png)